File: paper/idiomatic/rc.bio.jl
using BioSequences

function rcc(s)
   b = reverse_complement(s)
	println(String(b))
	length(b)
end

function main(arg) 
	f = open(arg)
	total = 0
	for l in eachline(f)
	   total += rcc(DNASequence(l))
	end
	println("$total")
end

main(ARGS[1])

File: paper/idiomatic/rc.bio.py
import sys
import Bio.Seq

def rc_copy(s):
   rc = s.reverse_complement()
   print rc
   return len(rc)

f = open(sys.argv[1], 'r')
total = 0
for l in f:
   total += rc_copy(Bio.Seq.Seq(l.strip()))
print total
